kolibri/core/kolibri_plugin.py

```diff
@@ -38,6 +38,7 @@ class KolibriCoreFrontEnd(KolibriFrontEndPluginBase):
     """
     entry_file = "assets/src/kolibri_core_app.js"
     external = True
+    core = True
```

Comment:
This 'core' property is only for defining a frontend KolibriModule as core. Should mention in the base class docs - if not, I'll add.